control/available_plugin.go

```diff
@@ -107,6 +107,9 @@ func newAvailablePlugin(resp plugin.Response, emitter gomit.Emitter, ep executab
 			}
 			ap.client = c
 		case plugin.NativeRPC:
+			log.WithFields(log.Fields{
```

Comment:
Give me the block also (newAvailablePlugin) so that I know exactly where this message came from.